clusterloader2/pkg/measurement/common/bundle/test_metrics.go

```diff
@@ -81,6 +81,9 @@ func createTestMetricsMeasurement() measurement.Measurement {
 	if metrics.systemPodMetrics, err = measurement.CreateMeasurement("SystemPodMetrics"); err != nil {
 		klog.Errorf("%v: systemPodMetrics creation error: %v", metrics, err)
 	}
+	if metrics.clusterOomsTracker, err = measurement.CreateMeasurement("ClusterOomsTracker"); err != nil {
```

Comment:
Please call it clusterOOMsTracker 

i.e. OOM should all be capital letters.

here and everywhere else (including doc)